doc/rtd/topics/tutorials/qemu.rst

```diff
@@ -0,0 +1,140 @@
+.. _tutorial_qemu:
+
+Qemu
+****
+
+In this tutorial, we will create our first cloud-init user data config and
+deploy it into a Qemu_ virtual machine. We'll be using Qemu for this tutorial
+which is a popular emulator for running virtual machines on Linux. Several
+popular virtual machine tools use Qemu, including Libvirt, LXD, and Vagrant.
+
+Install Qemu
+============
+
+.. code-block:: sh
+
+    $ sudo apt install qemu-system-x86
+
+See qemu's `install instructions <https://www.qemu.org/download/#linux>`_.
+
+Download a Cloud Image
+======================
+
+All commands should be executed from a single directory; some
```

Comment:
If it's generally considered a good idea to create a temporary directory, then for a tutorial doc this should be presented as an explicit step rather than an optional one. It wasn't completely clear to me if I had to download the image into the temp directory, so I didn't create one. It didn't occur to me until after that maybe I should have done that.

Reply:
+1